tests/e2e/cucumber/features/smoke/personalSpacePagination.feature

```diff
@@ -0,0 +1,24 @@
+Feature: check files pagination in personal space
+  As a user
+  I want to check the pagination of files
+  So that I can ensure files are displayed in proper pages
```

Comment:
Suggestion. what do you think?
```suggestion
  As a user
  I want to navigate a large number of files using pagination
  So that I do not have to scroll deep down
```